src/pages/courses/[slug]/[slug_lesson].tsx

```diff
@@ -0,0 +1,12 @@
+import { prisma } from '~/server/prisma';
```

Comment:
Question: why do we expose lessons publicly? Aren’t those for people who only buy the course?

Reply:
Yes, I have to after lock it but the issue is you can't fetch from FS using server side props

Reply:
I will always show blank before js loads basically

Reply:
Awesome let's 🚢